### PR TITLE
finish feature unit test

### PR DESCRIPTION
**Pull request by namdhgc**
@jackdiff Please take a review for me when you have free time

### CODE REVIEW

File: app/Http/Models/User.php [hunk 1]
```diff
@@ -17,6 +17,10 @@ class User extends Model
 
 	protected $table = 'users';
 
+	protected $fillable = [
+        'name', 'address', 'age', 'avatar_path'
+    ];
```
Comment (jackdiff) [inline app/Http/Models/User.php:22]: Please fix this indent

File: tests/Feature/UserControllerTest.php [hunk 2]
```diff
@@ -0,0 +1,569 @@
...
+                'name' 		=> $request_array['name'],
+                'age' 		=> $request_array['age'],
+                'address' 	=> $request_array['address'],
+            ]);
+        $response->assertJson( $base_response );
+	}
+
+	public function testAddUserFailWithConditionNameTooLong()
+	{
+		$Response 		= new Response();
```
Comment (jackdiff) [inline tests/Feature/UserControllerTest.php:43]: Please fix the indent

File: tests/Feature/UserControllerTest.php [hunk 3]
```diff
@@ -0,0 +1,569 @@
...
+            'name' 		=> 'abc',
+            'address' 	=> 'Test Dia Chi',
+            'age' 		=> 22,
+        ];
+        $delete_request_array = [
+            'id' 		=> 1,
+        ];
+        $Response 			= new Response();
+    	$base_response		= $Response->response( '200', Lang::get('message.web.success.0003'), $delete_request_array, true );
+        $response 			= $this->call('POST', 'user/add', $request_array);
```
Comment (jackdiff) [inline tests/Feature/UserControllerTest.php]: Should assert adding new user success here before do delete

File: tests/Feature/UserControllerTest.php [hunk 4]
```diff
@@ -0,0 +1,569 @@
...
+            'name' 		=> 'abc',
+            'address' 	=> 'Test Dia Chi',
+            'age' 		=> 22,
+        ];
+        $delete_request_array = [
+            'id' 		=> 2,
+        ];
+        $Response 			= new Response();
+    	$base_response		= $Response->response( '0001', Lang::get('message.web.error.0001'), "", false );
+        $response 			= $this->call('POST', 'user/add', $request_array);
```
Comment (jackdiff) [inline tests/Feature/UserControllerTest.php]: Should assert adding new user success here before do delete

File: tests/Feature/UserControllerTest.php [hunk 5]
```diff
@@ -0,0 +1,569 @@
...
+        $delete_response 	= $this->call('POST', 'user/delete', $delete_request_array);
```
Comment (jackdiff) [inline tests/Feature/UserControllerTest.php]: Should assert is there any record that has id = 2 in database before deleting

**Review by jackdiff — CHANGES_REQUESTED** [review]

File: app/Http/Models/User.php (hunk 1 above)
Comment (namdhgc) [reply app/Http/Models/User.php:22]: In my sublime text, indent looks good @@

File: tests/Feature/UserControllerTest.php (hunk 2 above)
Comment (namdhgc) [reply tests/Feature/UserControllerTest.php:43]: In my sublime text, indent looks good @@

File: tests/Feature/UserControllerTest.php (hunk 3 above)
Comment (namdhgc) [reply tests/Feature/UserControllerTest.php]: added assert adding newuser success before delete

File: tests/Feature/UserControllerTest.php (hunk 4 above)
Comment (namdhgc) [reply tests/Feature/UserControllerTest.php]: added assert adding new user success before delete

File: tests/Feature/UserControllerTest.php (hunk 5 above)
Comment (namdhgc) [reply tests/Feature/UserControllerTest.php]: added field id in assert is there any record has id = 2 before deleting

**Review by jackdiff — APPROVED** [review]

**Pull request merged**